main.go

```diff
@@ -24,12 +26,17 @@ func main() {
 	ts := oauth2.StaticTokenSource(&oauth2.Token{AccessToken: ghTkn})
 	cl := oauth2.NewClient(oauth2.NoContext, ts)
 	ghClient := github.NewClient(cl)
+	dockerCl, err := dlib.NewClientFromEnv()
```

Comment:
This should. The [code for this func](https://github.com/fsouza/go-dockerclient/blob/master/client.go#L237) uses the [`NewVersionedClientFromEnv`](https://github.com/fsouza/go-dockerclient#sourcegraph&def=GoPackage/github.com/fsouza/go-dockerclient/-/NewVersionedClientFromEnv) function under the covers, which in turn uses [`getDockerEnv()`](https://github.com/fsouza/go-dockerclient#sourcegraph&def=GoPackage/github.com/fsouza/go-dockerclient/-/getDockerEnv), which knows what to do if the `DOCKER_HOST` and related env vars are missing (as they would be in the case of docker for mac). I've tested the same command as above (`./deisrel docker tag v2.0.2 --prompt-tags=false --push=false --force-retag=true`) against a machine running Docker for Mac and saw it successfully connect to the docker daemon